kernel/src/fs/inode_handle/mod.rs

```diff
@@ -114,7 +114,7 @@ impl InodeHandle_ {
     }
 
     pub fn seek(&self, pos: SeekFrom) -> Result<usize> {
-        do_seek_util(self.path.inode(), &self.offset, pos)
```

Comment:
All methods on `InodeHandle_` should not have the `pub` visibility. Could you also cleanup the `pub` keyword in this PR?